packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -6,7 +6,15 @@ const ignoredA11y = {
 };
 
 describe('Time Input Twelve Hour Mobile', () => {
-  before(() => browser.setViewportSize(Terra.viewports('tiny')[0]));
+  before(() => {
+    browser.setViewportSize(Terra.viewports('tiny')[0]);
+    browser.execute(() => {
+      if (!window.ontouchstart) {
+        window.ontouchstart = 'true';
+      }
+      console.log('what the FFFFF');
```

Comment:
![https://media.giphy.com/media/xUA7aTaAgUK1qcv1WE/giphy.gif](https://media.giphy.com/media/xUA7aTaAgUK1qcv1WE/giphy.gif)

Reply:
lol, removed in this commit: https://github.com/cerner/terra-core/pull/2059/commits/d798696521cb0c5bf6e9d2550e21194b36dceb65